Source/Extensions/Blazorise.DataGrid/DataGrid.razor.cs

```diff
@@ -246,6 +246,12 @@ public override async Task SetParametersAsync( ParameterView parameters )
         {
             await CheckMultipleSelectionSetEmpty( parameters );
 
+            if ( parameters.TryGetValue<IEnumerable<TItem>>( nameof( Data ), out var changedData ) && !Data.IsEqual( changedData ) )

```

Comment:
Do we still want `.IsEqual` here, or `.AreEqual`, which is used only for lists?

Reply:
Honestly I forgot. From the tests it seemed to have the correct behaviour. I can change and retest.